Add DSPause

asymmetric opened the pull request:
Removes mom entirely.

The change in `.gitmodules` should be reverted before merging.

asymmetric committed:
update dss-deploy

asymmetric committed:
Track ds-pause branch of dss-deploy

asymmetric committed:
Use ds-pause, remove mom

asymmetric committed:
tests: Move spell election code to function

asymmetric committed:
Use dashes in error messages

gbalabasquer committed:
dapp upgrade dss-deploy from master branch

gbalabasquer reviewed (changes requested):
The `wait` param should be managed differently. As what `pause.plan` expects is the `era` time (which should be >= now + pause delay). Then in `schedule` you should be passing `now + wait` to `pause.plan`. You can't define `wait` as the `era` in the `line-spell` proposal because you do not know when it will be elected. Then you will also need some check so the `schedule` can be called just once.

asymmetric committed:
Point dss-deploy submodule back to master branch

asymmetric committed:
Pass delay at schedule time

It's not possible to calculate `now + delay` beforehand, since we don't know when
the spell will  be elected, so it makes less sense to pass the delay in the constructor.

asymmetric committed:
Retrieve delay at test runtime
